examples/mixed-multigrid-solver/mixed-multigrid-solver.cpp

```diff
@@ -79,20 +79,23 @@ int main(int argc, char* argv[])
              }},
             {"dpcpp",
              [] {
-                 return gko::DpcppExecutor::create(0,
-                                                   gko::OmpExecutor::create());
+                 return gko::DpcppExecutor::create(
+                     0, gko::ReferenceExecutor::create());
              }},
             {"reference", [] { return gko::ReferenceExecutor::create(); }}};
 
     // executor where Ginkgo will perform the computation
     const auto exec = exec_map.at(executor_string)();  // throws if not valid
-
+    const bool use_mixed = argc >= 5;
+    std::cout << "Use Mixed: " << use_mixed << std::endl;
```

Comment:
```suggestion
    std::cout << "Using mixed precision? " << use_mixed << std::endl;
```
Or maybe this was debug to begin with?